Scripts/WikiUpdater.cs

```diff
@@ -1,61 +1,58 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Text;
 using System.Text.Json.Serialization;
 using System.Text.RegularExpressions;
 using System.Threading;
 using System.Threading.Tasks;
+using AngleSharp.Dom;
 using AngleSharp.Html.Dom;
 using Scripts;
 using ScriptsBase.Models;
 using ScriptsBase.Utilities;
 
-public static class WikiUpdater
+public class WikiUpdater
 {
     private const string ORGANELLE_CATEGORY = "https://wiki.revolutionarygamesstudio.com/wiki/Category:Organelles";
     private const string WIKI_FILE = "simulation_parameters/common/wiki.json";
     private const string ENGLISH_TRANSLATION_FILE = "locale/en.po";
     private const string TEMP_TRANSLATION_FILE = "en.po.temp_wiki";
 
     /// <summary>
-    ///   Compounds to replace with custom BBCode when appearing in bold on wiki pages.
+    ///   List of regexes for domains we're allowing Thriveopedia content to link to.
     /// </summary>
-    private static readonly string[] CustomBbcodeCompounds =
-    {
-        // TODO: get these values from English translations of the names in organelles.json
-        "ATP",
-        "Ammonia",
-        "Carbon Dioxide",
-        "Glucose",
-        "Hydrogen Sulfide",
-        "Iron",
-        "Mucilage",
-        "Nitrogen",
-        "Oxygen",
-
-        // ReSharper disable once StringLiteralTypo
-        "OxyToxy",
-        "Phosphates",
-        "Sunlight",
-        "Temperature",
-    };
+    private readonly Regex[] whitelistedDomains =
```

Comment:
This could have stayed static, but as this class is basically instantiated only once ever when running a specific script, this is not going to matter.